BedrockServer.cpp

```diff
@@ -1665,18 +1678,23 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
 }
 
 unique_ptr<BedrockCommand> BedrockServer::getCommandFromPlugins(SData&& request) {
-    return getCommandFromPlugins(SQLiteCommand(move(request)));
+    return getCommandFromPlugins(make_unique<SQLiteCommand>(move(request)));
 }
 
-unique_ptr<BedrockCommand> BedrockServer::getCommandFromPlugins(SQLiteCommand&& baseCommand) {
+unique_ptr<BedrockCommand> BedrockServer::getCommandFromPlugins(unique_ptr<SQLiteCommand>&& baseCommand) {
     for (auto pair : plugins) {
-        auto command = pair.second->getCommand(move(baseCommand));
+
+        // This is a bit weird to avoid changing this signature in all the plugins. It would be more straightforward if
+        // the plugins just accepted a `unique_ptr<SQLiteCommand>&&`, but this still works.
+        auto command = pair.second->getCommand(move(*baseCommand));
```

Comment:
I'd prefer changing the `getCommand` signature to accept the `unique_ptr`, but that cacades through all the plugin repos.

Reply:
we can always do that in a follow up